src/system.cpp

```diff
@@ -8,6 +8,16 @@
 //-----------------------------------------------------------------------------
 #include "solvespace.h"
 
+#ifdef LIBRARY
```

Comment:
I am not happy with re-defining this here, but the `solvespace.h` header has to "undef" those because otherwise it conflicts with `Style::CONSTRAINT`